pyproject.toml

```diff
@@ -45,3 +45,35 @@ dev = [
 [build-system]
 requires = ["flit_core >=3.5,<4"]
 build-backend = "flit_core.buildapi"
+
+[tool.autopep8]
```

Comment:
(Maybe not in this cl) Can we use `pyink` instead ? This is becoming the Google default, a fork of black adapted to Google style. See: https://github.com/google/pyink

```toml
[tool.pyink]
# Formatting configuration to follow Google style-guide
pyink-indentation = 2
pyink-use-majority-quotes = true
```

And for vscode compatibility: https://github.com/google/pyink#is-there-a-vs-code-extension-for-pyink